syntax = "proto3";
package collect_face;

service CollectFace {
  rpc DoCollectFace(CollectFaceRequest) returns (CollectFaceResponse){}
}

message Image{
    bytes raw_data = 1;  //实时图片（raw_data）
    int32 width = 2;  //raw_data：宽
    int32 high = 3; //raw_data：高
    int32 channel = 4; //raw_data：通道（3）
}

message CollectFaceRequest {
    int32 staff_id = 1; //员工的ID(unique)
    }


message CollectFaceResponse{
    int32 staff_id = 1; //员工的ID(unique)
    Image image = 2;    //返回一张采集到的员工的人脸图像
    }